server/src/main/kotlin/io/spine/server/entity/extensions.kt

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2020, TeamDev. All rights reserved.
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
```

Comment:
It looks like an oldish disclaimer. Do you think we can start using the new one? It is available in master already and in the `config` master as well.

Reply:
This is so because the target branch `2.x-jdk8-master`, not `master` (with v1.7.0 in it).